addons/source-python/data/source-python/entities/csgo/CCSPlayer.ini

```diff
@@ -12,10 +12,10 @@ srv_check = False
 
     # Reference string: #Cannot_Carry_Anymore
     [[buy_internal]]
-        identifier_windows = 55 8B EC 83 E4 F8 83 EC 0C 53 56 57 6A 01
+        identifier_windows = 55 8B EC 83 E4 F8 83 EC 2C 53 56 57 6A 01
         # indentifier_linux = _ZN9CCSPlayer26HandleCommand_Buy_InternalEPKcib
         identifier_linux = 55 89 E5 83 EC 68 0F B6 55 14
```

Comment:
The signature of linux has also changed to "55 89 E5 83 EC 78 0F B6 55 14 89 75 F8".